2024-07-11 13:27:08,914 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 13:27:08,914 INFO:logging in using static token
2024-07-11 13:27:09,749 INFO:Shard ID None has connected to Gateway (Session ID: b994d437e37d3903acd932dd0c802fa2).
2024-07-11 17:17:37,794 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:17:37,795 INFO:logging in using static token
2024-07-11 17:17:38,539 INFO:Shard ID None has connected to Gateway (Session ID: a9378ea1ea380637689846d429b79f50).
2024-07-11 17:17:58,029 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:17:58,029 INFO:logging in using static token
2024-07-11 17:17:58,817 INFO:Shard ID None has connected to Gateway (Session ID: d92ab38996061aa358d07ad1be8d820b).
2024-07-11 17:18:03,445 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:18:03,446 INFO:logging in using static token
2024-07-11 17:18:04,298 INFO:Shard ID None has connected to Gateway (Session ID: 0184f77bd7f2c7182333bc428d8acbe7).
2024-07-11 17:18:13,562 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:18:13,562 INFO:logging in using static token
2024-07-11 17:18:14,457 INFO:Shard ID None has connected to Gateway (Session ID: 88a06bed1cd46482b1a274c50c80c128).
2024-07-11 17:18:16,764 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:18:16,764 INFO:logging in using static token
2024-07-11 17:18:17,509 INFO:Shard ID None has connected to Gateway (Session ID: 5b67906564ef4c7c1372dd1c48a602ef).
2024-07-11 17:18:21,932 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:18:21,932 INFO:logging in using static token
2024-07-11 17:18:22,643 INFO:Shard ID None has connected to Gateway (Session ID: 0e5bd7d41b1abb23cfbad9d753b60116).
2024-07-11 17:18:49,708 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:18:49,708 INFO:logging in using static token
2024-07-11 17:18:50,569 INFO:Shard ID None has connected to Gateway (Session ID: 9dcbc37d124c99fd29267ad303ab3f66).
2024-07-11 17:18:52,755 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 32, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 75, in winamax
    cache = json.load(file).get('winamax', [])
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
2024-07-11 17:18:53,999 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:18:53,999 INFO:logging in using static token
2024-07-11 17:18:54,758 INFO:Shard ID None has connected to Gateway (Session ID: 3a0b07a55354c78267b34b7f248baab8).
2024-07-11 17:18:56,924 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 75, in winamax
    cache = json.load(file).get('winamax', [])
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
2024-07-11 17:19:27,262 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:19:27,263 INFO:logging in using static token
2024-07-11 17:19:28,244 INFO:Shard ID None has connected to Gateway (Session ID: a0f5e60d2cd52fbf104b60561ce3c8a6).
2024-07-11 17:19:35,067 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:19:35,067 INFO:logging in using static token
2024-07-11 17:19:35,961 INFO:Shard ID None has connected to Gateway (Session ID: 63a74a537e7f4649a5e9dc1c96eb0ea9).
2024-07-11 17:19:38,102 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 75, in winamax
    cache = json.load(file).get('winamax', [])
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
2024-07-11 17:19:48,025 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:19:48,025 INFO:logging in using static token
2024-07-11 17:19:48,903 INFO:Shard ID None has connected to Gateway (Session ID: 1267ada60cba4d2eb1f126c4bd3d7794).
2024-07-11 17:20:01,800 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:20:01,801 INFO:logging in using static token
2024-07-11 17:20:02,571 INFO:Shard ID None has connected to Gateway (Session ID: 0d3f1ed11f39178429b1724adbb80090).
2024-07-11 17:20:10,354 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:20:10,355 INFO:logging in using static token
2024-07-11 17:20:11,669 INFO:Shard ID None has connected to Gateway (Session ID: cd5be2ec3b7ac32553aca68715b92a17).
2024-07-11 17:20:16,110 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:20:16,110 INFO:logging in using static token
2024-07-11 17:20:16,853 INFO:Shard ID None has connected to Gateway (Session ID: a8cf30f130b1fc79a5d6602284c479ea).
2024-07-11 17:20:19,012 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 75, in winamax
    cache = json.load(file).get('winamax', [])
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
2024-07-11 17:20:24,671 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:20:24,671 INFO:logging in using static token
2024-07-11 17:20:25,473 INFO:Shard ID None has connected to Gateway (Session ID: 079713daf0f55f5a8a2846c493b2d228).
2024-07-11 17:20:42,142 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:20:42,142 INFO:logging in using static token
2024-07-11 17:20:42,904 INFO:Shard ID None has connected to Gateway (Session ID: b70b1632d5f861f2934dc4f6544a7b22).
2024-07-11 17:20:48,914 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:20:48,914 INFO:logging in using static token
2024-07-11 17:20:49,943 INFO:Shard ID None has connected to Gateway (Session ID: 3510fb93fcc06a4d773d55247b1d8bb4).
2024-07-11 17:20:55,666 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:20:55,667 INFO:logging in using static token
2024-07-11 17:20:56,394 INFO:Shard ID None has connected to Gateway (Session ID: 64c3eb9b081dcd63dfe502429f3b5147).
2024-07-11 17:21:04,837 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:21:04,837 INFO:logging in using static token
2024-07-11 17:21:05,707 INFO:Shard ID None has connected to Gateway (Session ID: a8514ea099c857369d002ed473c8d20b).
2024-07-11 17:21:07,859 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 75, in winamax
    cache = json.load(file).get('winamax', [])
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
2024-07-11 17:21:10,782 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:21:10,782 INFO:logging in using static token
2024-07-11 17:21:11,743 INFO:Shard ID None has connected to Gateway (Session ID: 89e3e4867b447939557863602132a316).
2024-07-11 17:21:13,913 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 75, in winamax
    cache = json.load(file).get('winamax', [])
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
2024-07-11 17:21:21,095 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:21:21,095 INFO:logging in using static token
2024-07-11 17:21:21,925 INFO:Shard ID None has connected to Gateway (Session ID: 6ae8c2ba838139155cc80a20606c3d1b).
2024-07-11 17:21:24,126 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 74, in winamax
    print(ca@che_path)
NameError: name 'ca' is not defined
2024-07-11 17:21:31,104 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:21:31,104 INFO:logging in using static token
2024-07-11 17:21:32,378 INFO:Shard ID None has connected to Gateway (Session ID: eb5bad0cb059a8c53c933244831e1a30).
2024-07-11 17:21:34,561 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 76, in winamax
    cache = json.load(file).get('winamax', [])
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
2024-07-11 17:21:49,543 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:21:49,543 INFO:logging in using static token
2024-07-11 17:21:50,276 INFO:Shard ID None has connected to Gateway (Session ID: 870a6126d2a522f2af9948969ec42d01).
2024-07-11 17:21:52,451 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 76, in winamax
    cache = json.load(file).get('winamax', [])
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
2024-07-11 17:22:09,804 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:22:09,805 INFO:logging in using static token
2024-07-11 17:22:10,633 INFO:Shard ID None has connected to Gateway (Session ID: 60de21dedb4bf7e4edfbc19f0cc05ebb).
2024-07-11 17:22:12,803 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 76, in winamax
    cache = json.load(file).get('', [])
AttributeError: 'list' object has no attribute 'get'
2024-07-11 17:22:43,314 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:22:43,314 INFO:logging in using static token
2024-07-11 17:22:44,033 INFO:Shard ID None has connected to Gateway (Session ID: 5e34995ca8bafaad8fc2977fbf964f05).
2024-07-11 17:22:46,194 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 76, in winamax
    cache = json.load(file).get('', [])
AttributeError: 'list' object has no attribute 'get'
2024-07-11 17:23:02,578 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:23:02,579 INFO:logging in using static token
2024-07-11 17:23:03,343 INFO:Shard ID None has connected to Gateway (Session ID: 979ab9e7003f04fffb54118016b82ace).
2024-07-11 17:23:05,730 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 77, in winamax
    cache = json.load(file).get('', [])
AttributeError: 'list' object has no attribute 'get'
2024-07-11 17:23:11,626 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:23:11,626 INFO:logging in using static token
2024-07-11 17:23:12,317 INFO:Shard ID None has connected to Gateway (Session ID: 624789ee33187e49710b47b29dd0751c).
2024-07-11 17:23:14,509 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 77, in winamax
    cache = json.load(file).get('', [])
AttributeError: 'list' object has no attribute 'get'
2024-07-11 17:23:17,301 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:23:17,302 INFO:logging in using static token
2024-07-11 17:23:18,455 INFO:Shard ID None has connected to Gateway (Session ID: a1a62a8f39fe756dde8a2aaa4cd97297).
2024-07-11 17:23:20,618 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 76, in winamax
    cache = json.load(file).get('', [])
AttributeError: 'list' object has no attribute 'get'
2024-07-11 17:23:24,064 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:23:24,065 INFO:logging in using static token
2024-07-11 17:23:24,971 INFO:Shard ID None has connected to Gateway (Session ID: ba1e49a346153a17cf3d2472185bb108).
2024-07-11 17:23:27,135 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 76, in winamax
    cache = json.load(file).get('winamax', [])
AttributeError: 'list' object has no attribute 'get'
2024-07-11 17:23:30,944 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:23:30,944 INFO:logging in using static token
2024-07-11 17:23:31,878 INFO:Shard ID None has connected to Gateway (Session ID: e473ce269123ec68f293c032aa3139c4).
2024-07-11 17:23:34,049 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 77, in winamax
    print(cache)
UnboundLocalError: local variable 'cache' referenced before assignment
2024-07-11 17:23:42,239 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:23:42,240 INFO:logging in using static token
2024-07-11 17:23:43,116 INFO:Shard ID None has connected to Gateway (Session ID: a15618c0f8e5ff91b55047cde2d9bc09).
2024-07-11 17:23:45,266 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 77, in winamax
    print(cache)
UnboundLocalError: local variable 'cache' referenced before assignment
2024-07-11 17:24:00,108 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:24:00,108 INFO:logging in using static token
2024-07-11 17:24:01,234 INFO:Shard ID None has connected to Gateway (Session ID: fd2015b19a512a2e37d0864715028666).
2024-07-11 17:24:03,396 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 82, in winamax
    if boost not in cache:
UnboundLocalError: local variable 'cache' referenced before assignment
2024-07-11 17:24:13,273 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:24:13,274 INFO:logging in using static token
2024-07-11 17:24:14,256 INFO:Shard ID None has connected to Gateway (Session ID: 1ad9862cffc29d90737466c84f6dfce6).
2024-07-11 17:25:16,896 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:25:16,896 INFO:logging in using static token
2024-07-11 17:25:18,150 INFO:Shard ID None has connected to Gateway (Session ID: 135193778b3146e444baefadc012adfb).
2024-07-11 17:25:20,330 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 76, in winamax
    cache = json.load(file)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
2024-07-11 17:25:28,297 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:25:28,297 INFO:logging in using static token
2024-07-11 17:25:29,130 INFO:Shard ID None has connected to Gateway (Session ID: 0736785d0cf1d095a307d61ae985130b).
2024-07-11 17:25:56,373 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:25:56,373 INFO:logging in using static token
2024-07-11 17:25:57,271 INFO:Shard ID None has connected to Gateway (Session ID: f47e24ad1c91e700eca6eb17fca95c12).
2024-07-11 17:26:38,409 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:26:38,409 INFO:logging in using static token
2024-07-11 17:26:39,320 INFO:Shard ID None has connected to Gateway (Session ID: 0514f3dc6584a34a8288dcfc0d53f087).
2024-07-11 17:26:50,255 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:26:50,256 INFO:logging in using static token
2024-07-11 17:26:51,123 INFO:Shard ID None has connected to Gateway (Session ID: 0bd386d2a0213fb9b3dd690b036fec7a).
2024-07-11 17:27:34,431 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:27:34,431 INFO:logging in using static token
2024-07-11 17:27:35,596 INFO:Shard ID None has connected to Gateway (Session ID: 1a300ca880a6b9f41d6bcbeff8b51952).
2024-07-11 17:27:39,960 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:27:39,960 INFO:logging in using static token
2024-07-11 17:27:40,788 INFO:Shard ID None has connected to Gateway (Session ID: 677b61d9df291bbafe8583ed66d0c94d).
2024-07-11 17:28:13,765 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:28:13,766 INFO:logging in using static token
2024-07-11 17:28:14,743 INFO:Shard ID None has connected to Gateway (Session ID: ff236a3b8892517087ea7fba6e853d5a).
2024-07-11 17:28:19,862 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:28:19,862 INFO:logging in using static token
2024-07-11 17:28:20,600 INFO:Shard ID None has connected to Gateway (Session ID: b0646ba976c8ef3fe26fbae177104cb1).
2024-07-11 17:28:21,042 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:28:21,042 INFO:logging in using static token
2024-07-11 17:28:21,859 INFO:Shard ID None has connected to Gateway (Session ID: 1387a258be6426f564e42a0b0630ea4b).
2024-07-11 17:28:35,942 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:28:35,942 INFO:logging in using static token
2024-07-11 17:28:37,031 INFO:Shard ID None has connected to Gateway (Session ID: 2fba6347b5d284fbb3c273ce26278861).
2024-07-11 17:32:01,358 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:32:01,358 INFO:logging in using static token
2024-07-11 17:32:02,246 INFO:Shard ID None has connected to Gateway (Session ID: ab60f761d28f141e8dd4e9d4ccb57933).
2024-07-11 17:32:11,823 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:32:11,823 INFO:logging in using static token
2024-07-11 17:32:12,855 INFO:Shard ID None has connected to Gateway (Session ID: 96721e1f5c43646d380ff1a67dc1efbc).
2024-07-11 17:33:00,487 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:33:00,487 INFO:logging in using static token
2024-07-11 17:33:01,444 INFO:Shard ID None has connected to Gateway (Session ID: ce756d5b1a6bd9df7725b8cbe033c373).
2024-07-11 17:33:11,164 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:33:11,164 INFO:logging in using static token
2024-07-11 17:33:11,874 INFO:Shard ID None has connected to Gateway (Session ID: e17bde1ec850c97f3811ab396b0caa88).
2024-07-11 17:34:21,546 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:34:21,546 INFO:logging in using static token
2024-07-11 17:34:22,452 INFO:Shard ID None has connected to Gateway (Session ID: ab08b2eb27639d511bb8b8cc0c8fda2b).
2024-07-11 17:35:09,885 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:35:09,886 INFO:logging in using static token
2024-07-11 17:35:10,679 INFO:Shard ID None has connected to Gateway (Session ID: 4807bee205734510c7b6bcf000d03ddf).
2024-07-11 17:35:20,975 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:35:20,976 INFO:logging in using static token
2024-07-11 17:35:21,730 INFO:Shard ID None has connected to Gateway (Session ID: 22e27b9e4e81981a28b167c82c499891).
2024-07-11 17:35:35,773 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:35:35,774 INFO:logging in using static token
2024-07-11 17:35:36,595 INFO:Shard ID None has connected to Gateway (Session ID: 67c8261f50e87c9fd01bd2fbf0b819da).
2024-07-11 17:35:38,781 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 84, in winamax
    boost.delete('message_id', None)
AttributeError: 'dict' object has no attribute 'delete'
2024-07-11 17:35:52,542 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:35:52,542 INFO:logging in using static token
2024-07-11 17:35:53,250 INFO:Shard ID None has connected to Gateway (Session ID: e068ae838b63c73ee74f802544ecb8c2).
2024-07-11 17:36:43,932 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:36:43,933 INFO:logging in using static token
2024-07-11 17:36:44,761 INFO:Shard ID None has connected to Gateway (Session ID: 3e37a19b0ff1901d892658dce395353e).
2024-07-11 17:37:28,774 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:37:28,774 INFO:logging in using static token
2024-07-11 17:37:29,582 INFO:Shard ID None has connected to Gateway (Session ID: 47a5c334472be94307c6e8a095058487).
2024-07-11 17:38:20,115 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:38:20,115 INFO:logging in using static token
2024-07-11 17:38:21,003 INFO:Shard ID None has connected to Gateway (Session ID: 4be82ce20afa8e6fa3530b0d79d862b8).
2024-07-11 17:38:29,905 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:38:29,905 INFO:logging in using static token
2024-07-11 17:38:30,617 INFO:Shard ID None has connected to Gateway (Session ID: 109c72b3f0aaed2e207134fe3ad3b52a).
2024-07-11 17:38:47,871 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:38:47,871 INFO:logging in using static token
2024-07-11 17:38:48,694 INFO:Shard ID None has connected to Gateway (Session ID: 0857dac87b8917897cb7469478239322).
2024-07-11 17:38:59,108 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:38:59,108 INFO:logging in using static token
2024-07-11 17:38:59,933 INFO:Shard ID None has connected to Gateway (Session ID: f8c3ac436f1928d5424d040333c7b7a1).
2024-07-11 17:39:16,651 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:39:16,652 INFO:logging in using static token
2024-07-11 17:39:17,566 INFO:Shard ID None has connected to Gateway (Session ID: 0c56b79648632dba412a2b72fec33f87).
2024-07-11 17:39:19,741 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 107, in winamax
    message = await channel.fetch_message(boostCache['message_id'])
KeyError: 'message_id'
2024-07-11 17:40:05,571 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:40:05,572 INFO:logging in using static token
2024-07-11 17:40:06,255 INFO:Shard ID None has connected to Gateway (Session ID: 6cf4dd0c1b0c46b3f7b226c199b6d2ea).
2024-07-11 17:40:08,433 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 108, in winamax
    message = await channel.fetch_message(boostCache['message_id'])
KeyError: 'message_id'
2024-07-11 17:41:20,520 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:41:20,520 INFO:logging in using static token
2024-07-11 17:41:21,395 INFO:Shard ID None has connected to Gateway (Session ID: a709f8dc5c3fe14aa55ef04dd457f641).
2024-07-11 17:41:23,574 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 108, in winamax
    message = await channel.fetch_message(boostCache['message_id'])
KeyError: 'message_id'
2024-07-11 17:41:46,823 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:41:46,824 INFO:logging in using static token
2024-07-11 17:41:47,607 INFO:Shard ID None has connected to Gateway (Session ID: 3926f1e010bdbae21bcb3328bd353bc2).
2024-07-11 17:41:49,793 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 108, in winamax
    message = await channel.fetch_message(boostCache['message_id'])
KeyError: 'message_id'
2024-07-11 17:41:58,332 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:41:58,332 INFO:logging in using static token
2024-07-11 17:41:59,358 INFO:Shard ID None has connected to Gateway (Session ID: 5da8d140ba9d5bc5485e2e0c7de54ca4).
2024-07-11 17:42:26,050 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:42:26,050 INFO:logging in using static token
2024-07-11 17:42:26,910 INFO:Shard ID None has connected to Gateway (Session ID: 68b3b96c24d687353f6c21e1307db0f0).
2024-07-11 17:42:44,090 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 108, in winamax
    message = await channel.fetch_message(boostCache['message_id'])
KeyError: 'message_id'
2024-07-11 17:42:51,345 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:42:51,345 INFO:logging in using static token
2024-07-11 17:42:52,342 INFO:Shard ID None has connected to Gateway (Session ID: 9339ef10254e8086038901a6d0aeb66a).
2024-07-11 17:42:54,522 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 109, in winamax
    message = await channel.fetch_message(boostCache['message_id'])
KeyError: 'message_id'
2024-07-11 17:43:15,964 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:43:15,964 INFO:logging in using static token
2024-07-11 17:43:16,881 INFO:Shard ID None has connected to Gateway (Session ID: f5815febe9142ccc071b9d00dc03c73d).
2024-07-11 17:43:19,059 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 109, in winamax
    message = await channel.fetch_message(boostCache['message_id'])
KeyError: 'message_id'
2024-07-11 17:44:03,204 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:44:03,204 INFO:logging in using static token
2024-07-11 17:44:04,182 INFO:Shard ID None has connected to Gateway (Session ID: 2548a38282cdc7f55fcb573cd2aa489f).
2024-07-11 17:44:06,341 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 104, in winamax
    boost['message_id'] = boostCache['message_id']
KeyError: 'message_id'
2024-07-11 17:44:11,575 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:44:11,576 INFO:logging in using static token
2024-07-11 17:44:12,349 INFO:Shard ID None has connected to Gateway (Session ID: 8ddd3c66b6d79dcfada6af6537df9b30).
2024-07-11 17:44:30,061 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:44:30,061 INFO:logging in using static token
2024-07-11 17:44:31,077 INFO:Shard ID None has connected to Gateway (Session ID: d9db849a57800f8c39ff27daf530a959).
2024-07-11 17:44:48,423 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:44:48,423 INFO:logging in using static token
2024-07-11 17:44:49,102 INFO:Shard ID None has connected to Gateway (Session ID: a5a60e0866dfe80cc7a8a0daac7adb3c).
2024-07-11 17:44:58,414 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:44:58,415 INFO:logging in using static token
2024-07-11 17:44:59,114 INFO:Shard ID None has connected to Gateway (Session ID: cc05088f3064699844e889312b4fe730).
2024-07-11 17:45:02,582 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:45:02,582 INFO:logging in using static token
2024-07-11 17:45:03,413 INFO:Shard ID None has connected to Gateway (Session ID: 5f081c8492006eccfaa526f4e2ac5286).
2024-07-11 17:45:32,823 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:45:32,823 INFO:logging in using static token
2024-07-11 17:45:33,728 INFO:Shard ID None has connected to Gateway (Session ID: 16092cef5500387a3b85eda8fb4b9208).
2024-07-11 17:45:40,151 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:45:40,151 INFO:logging in using static token
2024-07-11 17:45:40,897 INFO:Shard ID None has connected to Gateway (Session ID: 0d5de013a30e25eab47eafa298551254).
2024-07-11 17:46:13,810 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:46:13,811 INFO:logging in using static token
2024-07-11 17:46:14,692 INFO:Shard ID None has connected to Gateway (Session ID: 0b97b55a2974dc7742045672cd98eec4).
2024-07-11 17:46:15,294 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:46:15,294 INFO:logging in using static token
2024-07-11 17:46:16,133 INFO:Shard ID None has connected to Gateway (Session ID: 2e12f88a15f92a529a5296d0a6dc21ca).
2024-07-11 17:46:35,508 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:46:35,508 INFO:logging in using static token
2024-07-11 17:46:36,262 INFO:Shard ID None has connected to Gateway (Session ID: a0e9d05886c4d2275337a84cc5a56e4f).
2024-07-11 17:48:02,102 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:48:02,102 INFO:logging in using static token
2024-07-11 17:48:02,955 INFO:Shard ID None has connected to Gateway (Session ID: 0f0c4d38de1c4ca7ac6e5f85c61c7c8b).
2024-07-11 17:48:10,080 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:48:10,081 INFO:logging in using static token
2024-07-11 17:48:11,062 INFO:Shard ID None has connected to Gateway (Session ID: 763b4950ff8feda521c2cd1fe0efe17a).
2024-07-11 17:48:16,029 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:48:16,029 INFO:logging in using static token
2024-07-11 17:48:16,885 INFO:Shard ID None has connected to Gateway (Session ID: 436143c886c6a74592b296bc914238c5).
2024-07-11 17:48:19,394 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 110, in winamax
    thread = await message.create_thread(name='Winamax Boosts', auto_archive_duration=60)
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/message.py", line 1407, in create_thread
    data = await self._state.http.start_thread_with_message(
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/http.py", line 758, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 400 Bad Request (error code: 160004): A thread has already been created for this message
2024-07-11 17:48:50,215 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:48:50,216 INFO:logging in using static token
2024-07-11 17:48:51,097 INFO:Shard ID None has connected to Gateway (Session ID: 9d18c2ce9a8f57a8a26afa2a2933d2b3).
2024-07-11 17:48:53,718 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 112, in winamax
    thread = await message.create_thread(name='Winamax Boosts', auto_archive_duration=60)
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/message.py", line 1407, in create_thread
    data = await self._state.http.start_thread_with_message(
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/http.py", line 758, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 400 Bad Request (error code: 160004): A thread has already been created for this message
2024-07-11 17:49:04,484 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:49:04,485 INFO:logging in using static token
2024-07-11 17:49:06,634 INFO:Shard ID None has connected to Gateway (Session ID: 1056f6fb2c3ad06c9c4ad56c0dad6dea).
2024-07-11 17:49:09,172 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 112, in winamax
    thread = await message.create_thread(name='Winamax Boosts', auto_archive_duration=60)
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/message.py", line 1407, in create_thread
    data = await self._state.http.start_thread_with_message(
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/http.py", line 758, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 400 Bad Request (error code: 160004): A thread has already been created for this message
2024-07-11 17:49:17,929 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:49:17,929 INFO:logging in using static token
2024-07-11 17:49:18,858 INFO:Shard ID None has connected to Gateway (Session ID: 546e239f93a22fa53858830d0bf1995e).
2024-07-11 17:49:30,452 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:49:30,452 INFO:logging in using static token
2024-07-11 17:49:31,177 INFO:Shard ID None has connected to Gateway (Session ID: 919528f10ab0c7ffe7c0545a5c4bae11).
2024-07-11 17:49:49,135 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 112, in winamax
    thread = await message.create_thread(name='Winamax Boosts', auto_archive_duration=60)
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/message.py", line 1407, in create_thread
    data = await self._state.http.start_thread_with_message(
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/http.py", line 758, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 400 Bad Request (error code: 160004): A thread has already been created for this message
2024-07-11 17:49:52,248 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:49:52,248 INFO:logging in using static token
2024-07-11 17:49:53,317 INFO:Shard ID None has connected to Gateway (Session ID: b409db53c7987ba656ad07c4ac2fff64).
2024-07-11 17:51:00,351 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:51:00,351 INFO:logging in using static token
2024-07-11 17:51:01,127 INFO:Shard ID None has connected to Gateway (Session ID: 7f2f2271e27520d9907ee2af16e99179).
2024-07-11 17:51:07,212 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:51:07,212 INFO:logging in using static token
2024-07-11 17:51:08,201 INFO:Shard ID None has connected to Gateway (Session ID: 5301aaff69bf7f64796a524e4dd3e1d5).
2024-07-11 17:51:10,772 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 112, in winamax
    await message.create_thread(name='Winamax Boosts', auto_archive_duration=60)
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/message.py", line 1407, in create_thread
    data = await self._state.http.start_thread_with_message(
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/http.py", line 758, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 400 Bad Request (error code: 160004): A thread has already been created for this message
2024-07-11 17:51:35,858 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:51:35,858 INFO:logging in using static token
2024-07-11 17:51:36,570 INFO:Shard ID None has connected to Gateway (Session ID: f52072e0a440875acedb6dd84047b17a).
2024-07-11 17:51:37,268 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:51:37,269 INFO:logging in using static token
2024-07-11 17:51:37,940 INFO:Shard ID None has connected to Gateway (Session ID: 7752074a30486af1254381fc87c76662).
2024-07-11 17:51:40,535 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 112, in winamax
    await message.create_thread(name='Winamax Boosts', auto_archive_duration=60)
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/message.py", line 1407, in create_thread
    data = await self._state.http.start_thread_with_message(
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/http.py", line 758, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 400 Bad Request (error code: 160004): A thread has already been created for this message
2024-07-11 17:51:45,380 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:51:45,380 INFO:logging in using static token
2024-07-11 17:51:46,260 INFO:Shard ID None has connected to Gateway (Session ID: 9ea8b1c7105066a08abf61228e3715ca).
2024-07-11 17:51:48,838 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 112, in winamax
    await message.create_thread(name='Winamax Boosts', auto_archive_duration=60)
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/message.py", line 1407, in create_thread
    data = await self._state.http.start_thread_with_message(
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/http.py", line 758, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 400 Bad Request (error code: 160004): A thread has already been created for this message
2024-07-11 17:52:13,920 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:52:13,920 INFO:logging in using static token
2024-07-11 17:52:14,745 INFO:Shard ID None has connected to Gateway (Session ID: 74f6cde2cec3eeb4067861eceac48eb3).
2024-07-11 17:52:17,277 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 114, in winamax
    await message.create_thread(name='Winamax Boosts', auto_archive_duration=60)
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/message.py", line 1407, in create_thread
    data = await self._state.http.start_thread_with_message(
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/http.py", line 758, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 400 Bad Request (error code: 160004): A thread has already been created for this message
2024-07-11 17:52:57,791 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:52:57,791 INFO:logging in using static token
2024-07-11 17:52:58,622 INFO:Shard ID None has connected to Gateway (Session ID: 863ec81aa5b43d415a6f15ee0cb1d8af).
2024-07-11 17:53:03,498 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:53:03,499 INFO:logging in using static token
2024-07-11 17:53:04,377 INFO:Shard ID None has connected to Gateway (Session ID: eeb0300a58c8b02eb0ac68035629b87f).
2024-07-11 17:53:08,644 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:53:08,645 INFO:logging in using static token
2024-07-11 17:53:09,497 INFO:Shard ID None has connected to Gateway (Session ID: 99b808c79b7f76a36f70f49c0d2e67c2).
2024-07-11 17:53:12,017 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 115, in winamax
    await thread_channel.send(embed=embed)
UnboundLocalError: local variable 'embed' referenced before assignment
2024-07-11 17:53:16,873 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:53:16,873 INFO:logging in using static token
2024-07-11 17:53:17,696 INFO:Shard ID None has connected to Gateway (Session ID: a64950a8ee54b08efa6dca7d8e6fb7b3).
2024-07-11 17:53:28,257 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:53:28,257 INFO:logging in using static token
2024-07-11 17:53:29,067 INFO:Shard ID None has connected to Gateway (Session ID: aac45303cc636f3c1d96a38485707f81).
2024-07-11 17:53:46,397 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:53:46,397 INFO:logging in using static token
2024-07-11 17:53:47,133 INFO:Shard ID None has connected to Gateway (Session ID: 2a9f7b71403c3eb3669703a310147197).
2024-07-11 17:53:53,116 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:53:53,116 INFO:logging in using static token
2024-07-11 17:53:54,014 INFO:Shard ID None has connected to Gateway (Session ID: 99bc1a7ef268312fc064c0eb5c51bdba).
2024-07-11 17:54:05,648 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:54:05,648 INFO:logging in using static token
2024-07-11 17:54:06,372 INFO:Shard ID None has connected to Gateway (Session ID: 5a4c3baf22bf0585a3bc9125d32351bc).
2024-07-11 17:54:12,430 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:54:12,430 INFO:logging in using static token
2024-07-11 17:54:13,413 INFO:Shard ID None has connected to Gateway (Session ID: e8213b6dddccc8869609a9738e3d6cfc).
2024-07-11 17:54:31,393 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:54:31,393 INFO:logging in using static token
2024-07-11 17:54:32,610 INFO:Shard ID None has connected to Gateway (Session ID: bd2ce7ad0d66dd0943850b80376e0d79).
2024-07-11 17:54:38,349 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:54:38,349 INFO:logging in using static token
2024-07-11 17:54:39,617 INFO:Shard ID None has connected to Gateway (Session ID: b4b0d84cad840e8d4348b9c808089513).
2024-07-11 17:54:42,159 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 115, in winamax
    await thread.send(embed=embed)
UnboundLocalError: local variable 'embed' referenced before assignment
2024-07-11 17:55:55,124 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:55:55,124 INFO:logging in using static token
2024-07-11 17:55:55,845 INFO:Shard ID None has connected to Gateway (Session ID: caa9760167c369cb6e9467991f3ccfe3).
2024-07-11 17:56:09,746 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:56:09,746 INFO:logging in using static token
2024-07-11 17:56:10,532 INFO:Shard ID None has connected to Gateway (Session ID: 9b505ebb0b590a3c40c92222c132d1dd).
2024-07-11 17:56:13,153 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 115, in winamax
    await thread.send(embed=embed)
UnboundLocalError: local variable 'embed' referenced before assignment
2024-07-11 17:56:38,884 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:56:38,884 INFO:logging in using static token
2024-07-11 17:56:39,583 INFO:Shard ID None has connected to Gateway (Session ID: 79d7c2e8e8c488326cf126fb5ff9e285).
2024-07-11 17:57:21,026 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:57:21,027 INFO:logging in using static token
2024-07-11 17:57:21,760 INFO:Shard ID None has connected to Gateway (Session ID: 714a4f329309bea334b6e9596aef6657).
2024-07-11 17:57:25,725 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:57:25,726 INFO:logging in using static token
2024-07-11 17:57:26,675 INFO:Shard ID None has connected to Gateway (Session ID: 7d3da94e181301d0448e89f950867f9d).
2024-07-11 17:58:05,065 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:58:05,065 INFO:logging in using static token
2024-07-11 17:58:05,811 INFO:Shard ID None has connected to Gateway (Session ID: 20a18dd3d490a34ce6da5f424f31f7ac).
2024-07-11 17:58:14,255 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:58:14,255 INFO:logging in using static token
2024-07-11 17:58:15,288 INFO:Shard ID None has connected to Gateway (Session ID: b3702980a6615342dce6a2c9347218a3).
2024-07-11 17:58:17,646 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 124, in winamax
    await thread.send(embed=embed)
UnboundLocalError: local variable 'thread' referenced before assignment
2024-07-11 17:58:36,692 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:58:36,693 INFO:logging in using static token
2024-07-11 17:58:37,532 INFO:Shard ID None has connected to Gateway (Session ID: 8ee6b0fc6b15e8831d52f48cbac9a464).
2024-07-11 17:58:39,897 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 110, in winamax
    if not boostCache['thread_id']:
KeyError: 'thread_id'
2024-07-11 17:58:52,623 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:58:52,624 INFO:logging in using static token
2024-07-11 17:58:53,318 INFO:Shard ID None has connected to Gateway (Session ID: 546af1ec95b9c1f42ecf00514d642888).
2024-07-11 17:58:55,690 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 110, in winamax
    if not boostCache['thread_id']:
KeyError: 'thread_id'
2024-07-11 17:59:00,393 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:59:00,393 INFO:logging in using static token
2024-07-11 17:59:01,224 INFO:Shard ID None has connected to Gateway (Session ID: 546e4120e35278c3182ff6de1e6301c8).
2024-07-11 17:59:03,587 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 110, in winamax
    if not boostCache['thread_id']:
KeyError: 'thread_id'
2024-07-11 17:59:13,651 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 17:59:13,651 INFO:logging in using static token
2024-07-11 17:59:14,514 INFO:Shard ID None has connected to Gateway (Session ID: 6231a9c0bb736cd9e3c6ca268fb0a67c).
2024-07-11 18:00:10,880 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:00:10,880 INFO:logging in using static token
2024-07-11 18:00:11,780 INFO:Shard ID None has connected to Gateway (Session ID: 0cc020c20cd3f2b46716ae0fe547f91a).
2024-07-11 18:00:18,204 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:00:18,204 INFO:logging in using static token
2024-07-11 18:00:19,124 INFO:Shard ID None has connected to Gateway (Session ID: 98c8049315c80fea72a6a8f9435f839b).
2024-07-11 18:00:21,484 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 116, in winamax
    await thread.send('Nouvelle cote boostée !', embed=embed)
UnboundLocalError: local variable 'embed' referenced before assignment
2024-07-11 18:01:09,843 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:01:09,846 INFO:logging in using static token
2024-07-11 18:01:10,661 INFO:Shard ID None has connected to Gateway (Session ID: 5c62a3323e0af7bfe21a65b65e2c4de4).
2024-07-11 18:01:26,105 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:01:26,105 INFO:logging in using static token
2024-07-11 18:01:26,924 INFO:Shard ID None has connected to Gateway (Session ID: 89a75f3af40f20d1e95f2e96062039b6).
2024-07-11 18:01:44,410 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 111, in winamax
    thread = await message.create_thread(name=f'Thread du boost {boostCache["intitule"]}', auto_archive_duration=60)
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/message.py", line 1407, in create_thread
    data = await self._state.http.start_thread_with_message(
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/http.py", line 758, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 400 Bad Request (error code: 50035): Invalid Form Body
In name: Must be between 1 and 100 in length.
2024-07-11 18:01:57,122 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:01:57,122 INFO:logging in using static token
2024-07-11 18:01:58,208 INFO:Shard ID None has connected to Gateway (Session ID: 9645a8feae96d3b0744f4ecdd5b418d3).
2024-07-11 18:02:00,682 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 111, in winamax
    thread = await message.create_thread(name=f'Thread du boost {boostCache["intitule"]}', auto_archive_duration=60)
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/message.py", line 1407, in create_thread
    data = await self._state.http.start_thread_with_message(
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/http.py", line 758, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 400 Bad Request (error code: 50035): Invalid Form Body
In name: Must be between 1 and 100 in length.
2024-07-11 18:02:57,900 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:02:57,900 INFO:logging in using static token
2024-07-11 18:02:58,720 INFO:Shard ID None has connected to Gateway (Session ID: c293c4be2c5709f3710a92c68fa7dc54).
2024-07-11 18:04:38,249 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:04:38,249 INFO:logging in using static token
2024-07-11 18:04:39,249 INFO:Shard ID None has connected to Gateway (Session ID: cac1214f81dd6556975f36b4a6be2bcb).
2024-07-11 18:04:48,789 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:04:48,789 INFO:logging in using static token
2024-07-11 18:04:49,671 INFO:Shard ID None has connected to Gateway (Session ID: 1b35f6edcba317a6c2ebbeaf8f62ef5a).
2024-07-11 18:04:52,812 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/boosts/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/boosts/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/boosts/boosts/winamax/script.py", line 116, in winamax
    await thread.send('Le boost a été modifié !', embed=embed)
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/abc.py", line 1618, in send
    data = await state.http.send_message(channel.id, params=params)
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/http.py", line 754, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10003): Unknown Channel
2024-07-11 18:07:30,860 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:07:30,860 INFO:logging in using static token
2024-07-11 18:07:31,683 INFO:Shard ID None has connected to Gateway (Session ID: b541fcf3e9952c63d7992186a67fa189).
2024-07-11 18:07:33,952 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:07:33,952 INFO:logging in using static token
2024-07-11 18:07:34,934 INFO:Shard ID None has connected to Gateway (Session ID: 0254a30098cfcc35b29cc55d7cfe87fb).
2024-07-11 18:07:37,243 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/elcrackito93/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/elcrackito93/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/elcrackito93/boosts/winamax/script.py", line 108, in winamax
    message = await channel.fetch_message(boostCache['message_id'])
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/abc.py", line 1684, in fetch_message
    data = await self._state.http.get_message(channel.id, id)
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/http.py", line 754, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10008): Unknown Message
2024-07-11 18:07:47,218 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:07:47,218 INFO:logging in using static token
2024-07-11 18:07:48,173 INFO:Shard ID None has connected to Gateway (Session ID: 629da5325d51c5566de5d3dcf3403ae3).
2024-07-11 18:07:52,054 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/elcrackito93/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/elcrackito93/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/elcrackito93/boosts/winamax/script.py", line 108, in winamax
    message = await channel.fetch_message(boostCache['message_id'])
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/abc.py", line 1684, in fetch_message
    data = await self._state.http.get_message(channel.id, id)
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/http.py", line 754, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10008): Unknown Message
2024-07-11 18:08:11,782 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:08:11,782 INFO:logging in using static token
2024-07-11 18:08:12,687 INFO:Shard ID None has connected to Gateway (Session ID: 9850a8e80abc639f86186a382176f017).
2024-07-11 18:08:15,223 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/elcrackito93/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/elcrackito93/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/elcrackito93/boosts/winamax/script.py", line 108, in winamax
    message = await channel.fetch_message(boostCache['message_id'])
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/abc.py", line 1684, in fetch_message
    data = await self._state.http.get_message(channel.id, id)
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/http.py", line 754, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10008): Unknown Message
2024-07-11 18:08:32,494 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:08:32,494 INFO:logging in using static token
2024-07-11 18:08:33,259 INFO:Shard ID None has connected to Gateway (Session ID: 64428da06c3def9faf280ae16722f57b).
2024-07-11 18:09:20,280 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:09:20,281 INFO:logging in using static token
2024-07-11 18:09:21,169 INFO:Shard ID None has connected to Gateway (Session ID: a8592205e6238b01fba802b252cb6dfc).
2024-07-11 18:09:24,472 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/elcrackito93/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/elcrackito93/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/elcrackito93/boosts/winamax/script.py", line 118, in winamax
    await thread.send('Le boost a été modifié !', embed=embed)
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/abc.py", line 1618, in send
    data = await state.http.send_message(channel.id, params=params)
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/http.py", line 754, in request
    raise NotFound(response, data)
discord.errors.NotFound: 404 Not Found (error code: 10003): Unknown Channel
2024-07-11 18:09:28,733 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:09:28,734 INFO:logging in using static token
2024-07-11 18:09:30,797 INFO:Shard ID None has connected to Gateway (Session ID: a123112750941e6e3dba36e4c7e7e576).
2024-07-11 18:09:32,955 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/elcrackito93/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/elcrackito93/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/elcrackito93/boosts/winamax/script.py", line 108, in winamax
    message = await channel.fetch_message(boostCache['message_id'])
AttributeError: 'NoneType' object has no attribute 'fetch_message'
2024-07-11 18:09:49,359 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:09:49,360 INFO:logging in using static token
2024-07-11 18:09:50,308 INFO:Shard ID None has connected to Gateway (Session ID: 5be09715ccc30cad05bf8ddc0185143c).
2024-07-11 18:11:30,404 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:11:30,404 INFO:logging in using static token
2024-07-11 18:11:31,163 INFO:Shard ID None has connected to Gateway (Session ID: fb3abc430466b03e8c9f142c9c4a69ea).
2024-07-11 18:11:32,420 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:11:32,421 INFO:logging in using static token
2024-07-11 18:11:33,221 INFO:Shard ID None has connected to Gateway (Session ID: 160e313a8ded189214627a1eb23646db).
2024-07-11 18:11:37,558 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:11:37,559 INFO:logging in using static token
2024-07-11 18:11:38,363 INFO:Shard ID None has connected to Gateway (Session ID: 6a159d9a4d45e3d1d54928296ee337aa).
2024-07-11 18:11:55,540 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:11:55,540 INFO:logging in using static token
2024-07-11 18:11:56,474 INFO:Shard ID None has connected to Gateway (Session ID: 1a051cf365c1ad79f8bf2c4f2dc87eb8).
2024-07-11 18:12:07,496 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:12:07,496 INFO:logging in using static token
2024-07-11 18:12:08,212 INFO:Shard ID None has connected to Gateway (Session ID: 5789f377633878c0e5b8eb31cb034bcf).
2024-07-11 18:12:21,865 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:12:21,865 INFO:logging in using static token
2024-07-11 18:12:22,803 INFO:Shard ID None has connected to Gateway (Session ID: ba92cb470c3b1e0af20ac81b2d000f8c).
2024-07-11 18:13:00,035 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:13:00,035 INFO:logging in using static token
2024-07-11 18:13:00,823 INFO:Shard ID None has connected to Gateway (Session ID: ec60cd5f8f5f6a6ec8f11d52d6d425e1).
2024-07-11 18:13:46,288 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:13:46,289 INFO:logging in using static token
2024-07-11 18:13:47,186 INFO:Shard ID None has connected to Gateway (Session ID: f0426da6de7d386ad184c850d4fe0262).
2024-07-11 18:13:54,654 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:13:54,654 INFO:logging in using static token
2024-07-11 18:13:55,524 INFO:Shard ID None has connected to Gateway (Session ID: 57010554ff15956258760a3a3621bdc4).
2024-07-11 18:15:32,364 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:15:32,365 INFO:logging in using static token
2024-07-11 18:15:33,245 INFO:Shard ID None has connected to Gateway (Session ID: ae661d6c1977374a16c4d0031375e79b).
2024-07-11 18:15:47,901 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:15:47,901 INFO:logging in using static token
2024-07-11 18:15:48,694 INFO:Shard ID None has connected to Gateway (Session ID: 504fbccf2b0e6759e7a05045796a9668).
2024-07-11 18:16:00,084 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:16:00,084 INFO:logging in using static token
2024-07-11 18:16:00,766 INFO:Shard ID None has connected to Gateway (Session ID: 9173dacf9f781f1a003bfa406e6dff15).
2024-07-11 18:16:03,263 ERROR:Unhandled exception in internal background task 'boosts'.
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/ext/tasks/__init__.py", line 241, in _loop
    await self.coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/elcrackito93/main.py", line 31, in boosts
    await search_boosts(bot)
  File "/Users/louwis/luis/bot/discord/elcrackito93/boosts/utils.py", line 7, in search_boosts
    await winamax(bot, cache_path);
  File "/Users/louwis/luis/bot/discord/elcrackito93/boosts/winamax/script.py", line 111, in winamax
    thread = await message.create_thread(name=f'Un boost a été modifié !', auto_archive_duration=1)
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/message.py", line 1407, in create_thread
    data = await self._state.http.start_thread_with_message(
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/http.py", line 758, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 400 Bad Request (error code: 50035): Invalid Form Body
In auto_archive_duration: Value "1" is not a valid enum value.
2024-07-11 18:16:14,006 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:16:14,006 INFO:logging in using static token
2024-07-11 18:16:14,776 INFO:Shard ID None has connected to Gateway (Session ID: 9844bb3b9717d7762c51048398476fc3).
2024-07-11 18:17:50,786 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:17:50,786 INFO:logging in using static token
2024-07-11 18:17:51,626 INFO:Shard ID None has connected to Gateway (Session ID: bd0967dd6703b327fb049309c420e299).
2024-07-11 18:18:21,537 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:18:21,537 INFO:logging in using static token
2024-07-11 18:18:22,401 INFO:Shard ID None has connected to Gateway (Session ID: b0b29056217e813dd7b3e7b5120a14b5).
2024-07-11 18:18:24,763 ERROR:Ignoring exception in on_message
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/client.py", line 449, in _run_event
    await coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/elcrackito93/main.py", line 35, in on_message
    await bot.process_commands(message)
AttributeError: 'Client' object has no attribute 'process_commands'
2024-07-11 18:18:25,116 ERROR:Ignoring exception in on_message
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/client.py", line 449, in _run_event
    await coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/elcrackito93/main.py", line 35, in on_message
    await bot.process_commands(message)
AttributeError: 'Client' object has no attribute 'process_commands'
2024-07-11 18:18:25,361 ERROR:Ignoring exception in on_message
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/client.py", line 449, in _run_event
    await coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/elcrackito93/main.py", line 35, in on_message
    await bot.process_commands(message)
AttributeError: 'Client' object has no attribute 'process_commands'
2024-07-11 18:18:25,609 ERROR:Ignoring exception in on_message
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/client.py", line 449, in _run_event
    await coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/elcrackito93/main.py", line 35, in on_message
    await bot.process_commands(message)
AttributeError: 'Client' object has no attribute 'process_commands'
2024-07-11 18:18:25,852 ERROR:Ignoring exception in on_message
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/client.py", line 449, in _run_event
    await coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/elcrackito93/main.py", line 35, in on_message
    await bot.process_commands(message)
AttributeError: 'Client' object has no attribute 'process_commands'
2024-07-11 18:18:26,098 ERROR:Ignoring exception in on_message
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/client.py", line 449, in _run_event
    await coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/elcrackito93/main.py", line 35, in on_message
    await bot.process_commands(message)
AttributeError: 'Client' object has no attribute 'process_commands'
2024-07-11 18:18:31,037 ERROR:Ignoring exception in on_message
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/client.py", line 449, in _run_event
    await coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/elcrackito93/main.py", line 35, in on_message
    await bot.process_commands(message)
AttributeError: 'Client' object has no attribute 'process_commands'
2024-07-11 18:18:31,260 ERROR:Ignoring exception in on_message
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/client.py", line 449, in _run_event
    await coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/elcrackito93/main.py", line 35, in on_message
    await bot.process_commands(message)
AttributeError: 'Client' object has no attribute 'process_commands'
2024-07-11 18:18:40,001 ERROR:Ignoring exception in on_message
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/client.py", line 449, in _run_event
    await coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/elcrackito93/main.py", line 35, in on_message
    await bot.process_commands(message)
AttributeError: 'Client' object has no attribute 'process_commands'
2024-07-11 18:18:40,236 ERROR:Ignoring exception in on_message
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/client.py", line 449, in _run_event
    await coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/elcrackito93/main.py", line 35, in on_message
    await bot.process_commands(message)
AttributeError: 'Client' object has no attribute 'process_commands'
2024-07-11 18:18:40,239 ERROR:Ignoring exception in on_message
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/client.py", line 449, in _run_event
    await coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/elcrackito93/main.py", line 35, in on_message
    await bot.process_commands(message)
AttributeError: 'Client' object has no attribute 'process_commands'
2024-07-11 18:18:55,056 ERROR:Ignoring exception in on_message
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/client.py", line 449, in _run_event
    await coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/elcrackito93/main.py", line 35, in on_message
    await bot.process_commands(message)
AttributeError: 'Client' object has no attribute 'process_commands'
2024-07-11 18:18:55,355 ERROR:Ignoring exception in on_message
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/client.py", line 449, in _run_event
    await coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/elcrackito93/main.py", line 35, in on_message
    await bot.process_commands(message)
AttributeError: 'Client' object has no attribute 'process_commands'
2024-07-11 18:18:55,452 ERROR:Ignoring exception in on_message
Traceback (most recent call last):
  File "/Users/louwis/Library/Python/3.9/lib/python/site-packages/discord/client.py", line 449, in _run_event
    await coro(*args, **kwargs)
  File "/Users/louwis/luis/bot/discord/elcrackito93/main.py", line 35, in on_message
    await bot.process_commands(message)
AttributeError: 'Client' object has no attribute 'process_commands'
2024-07-11 18:19:12,081 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:19:12,081 INFO:logging in using static token
2024-07-11 18:19:12,796 INFO:Shard ID None has connected to Gateway (Session ID: 03af64e9074b72b4ffa818585c655d42).
2024-07-11 18:19:20,922 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:19:20,922 INFO:logging in using static token
2024-07-11 18:19:21,907 INFO:Shard ID None has connected to Gateway (Session ID: 68265d6c3168d64600a89c8fc8db42fb).
2024-07-11 18:19:52,628 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:19:52,628 INFO:logging in using static token
2024-07-11 18:19:53,551 INFO:Shard ID None has connected to Gateway (Session ID: c45f42fd479c246640351863eb381e89).
2024-07-11 18:20:15,510 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:20:15,510 INFO:logging in using static token
2024-07-11 18:20:16,489 INFO:Shard ID None has connected to Gateway (Session ID: b678220b9250dfbb95c29bd53ef24f70).
2024-07-11 18:20:42,949 WARNING:PyNaCl is not installed, voice will NOT be supported
2024-07-11 18:20:42,949 INFO:logging in using static token
2024-07-11 18:20:43,716 INFO:Shard ID None has connected to Gateway (Session ID: c3439249bf57c73b9cdf405f416dc473).
